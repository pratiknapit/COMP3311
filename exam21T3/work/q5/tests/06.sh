python3 ./q5 Apartment 850000 0 0 0
