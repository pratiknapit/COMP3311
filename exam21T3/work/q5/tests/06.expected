#46428: 5/74 Leonard Avenue, Kingsford 2032, 1br, 1ba, 0car, $620000
#45874: 13/94 Jellicoe Avenue, Kingsford 2032, 1br, 1ba, 1car, $647000
#46784: 7/38 Borrodale Road, Kingsford 2032, 1br, 1ba, 0car, $695000
#45939: 20/79 Meeks Street, Kingsford 2032, 1br, 2ba, 0car, $701000
#46454: 14/100 Cooper Street, Kingsford 2032, 1br, 1ba, 0car, $720000
#46660: 18/27 Shaw Avenue, Kingsford 2032, 1br, 2ba, 0car, $761000
#46652: 12/42 Beulah Street, Kingsford 2032, 1br, 2ba, 1car, $775000
#46322: 20/88 Avoca Street, Kingsford 2032, 1br, 1ba, 1car, $779000
#46423: 18/9 Smith Street, Kingsford 2032, 2br, 1ba, 1car, $786000
#46410: 9/27 Burragulung Street, Kingsford 2032, 1br, 1ba, 0car, $812000
#46421: 14/48 Walenore Avenue, Kingsford 2032, 1br, 2ba, 0car, $819000
#46471: 15/71 Strachan Street, Kingsford 2032, 2br, 2ba, 1car, $823000
#46092: 11/46 Hincks Street, Kingsford 2032, 1br, 2ba, 1car, $829000
#46620: 9/5 Walenore Avenue, Kingsford 2032, 2br, 2ba, 0car, $832000
#46712: 15/5 Forsyth Street, Kingsford 2032, 2br, 2ba, 0car, $834000
#46278: 12/65 Broadbent Street, Kingsford 2032, 3br, 1ba, 1car, $845000
#46106: 5/87 Fischer Street, Kingsford 2032, 3br, 2ba, 0car, $847000
